src/components/views/read/cvForm/References.tsx

```diff
@@ -26,31 +26,26 @@ interface Props {
   nextStep: () => void;
   prevStep: () => void;
   cvValues: CVInfoObj;
```

Comment:
I guess this is now unused, we should remove it.

Reply:
But if we removed it, the user won't be able to back to a certain section right? "same comment for the below edit required"